.swiftlint.yml

```diff
@@ -10,6 +10,7 @@ excluded:
   - Build
   - .build
   - Tests
+  - AdyenCardScannerTests
```

Comment:
Isn't `Tests` covering the whole test suite?

Reply:
For the AdyenCardScanner module, I decided to have its own target/folder within our project.